config/prow/config.yaml

```diff
@@ -727,6 +727,31 @@ presets:
   - name: azure-ssh
     mountPath: /etc/azure-ssh
     readOnly: true
+- labels:
+    preset-azure-cred-anonymous-pull: "true"
+  env:
+    - name: AZURE_CREDENTIALS
+      value: /etc/azure-cred/credentials
+    - name: AZURE_SSH_PUBLIC_KEY_FILE
+      value: /etc/azure-ssh/azure-ssh-pub
+    - name: REGISTRY
+      value: capzci.azurecr.io
+    - name: AZ_STORAGE_CONTAINER_NAME
```

Comment:
do we need this?

Reply:
I don't think so. 

@chewong is this in use anywhere in CAPZ related jobs?

Reply:
I removed the storage bucket and broke out `preset-azure-anonymous-pull` and `preset-azure-cred-only` which will allow easier composition going forward.

Reply:
Only `AZURE_CREDENTIALS` is used by CAPZ e2e test. We might not even need `AZURE_SSH_PUBLIC_KEY_FILE` and allow the script to generate one during test run.